optimum/exporters/onnx/base.py

```diff
@@ -435,7 +435,7 @@ def ordered_inputs(self, model: Union["PreTrainedModel", "TFPreTrainedModel"]) -
             sig = inspect.signature(model.call)
 
         for param in sig.parameters:
-            param_regex = re.compile(rf"{param}(\.\d*)?")
+            param_regex = re.compile(rf"{param}(\..*)?$")
```

Comment:
this modification comes from `timestep` matching both `timestep` and `timestep_cond` previously (behavior that we don't want), we still want `past_key_value` to match `past_key_values.0.key` though